reference/docs-conceptual/getting-started/cookbooks/Decode-PowerShell-Command-from-a-Running-Process.md

```diff
@@ -0,0 +1,92 @@
+# Decode PowerShell Command from a Running Process
+
+There are times when I've found a PowerShell process running that is taking up a bunch of resources. Sometimes they've even been my own scripts running in the context of a [Task Scheduler](https://docs.microsoft.com/windows/desktop/TaskSchd/task-scheduler-start-page
+) job or a [SQL Server Agent](https://docs.microsoft.com/sql/ssms/agent/sql-server-agent
+) Job with a PowerShell job step. Because multiple PowerShell processes can be running at a time, sometimes it's difficult to know what the offending process is. The following method can be used to decode a script block that a PowerShell process is currently running.
+
+## Create a Long Running Process
+
+To demonstrate this capability, open a new PowerShell window and run the following code. It executes a PowerShell command that outputs a number every minute for 10 minutes.
+
+```PowerShell
+powershell.exe -Command {
+    $i = 1
+    while ( $i -le 10 )
+    {
+        Write-Output -InputObject $i
+        Start-Sleep -Seconds 60
+        $i++
+    }
+}
+```
+
+## View the Process
+
+### Using Task Manager
```

Comment:
Skip the task manager example for several reasons:
- It is Windows specific and we should be more platform agnostic
- It avoids the need for screenshots
- The focus should be on finding encoded commands; not looking at performance

Reply:
Done.